docs/manual/forms/logic_dmn.rst

```diff
@@ -0,0 +1,55 @@
+.. _manual_logic_dmn:
+
+====================
+Beslistabellen (DMN)
+====================
+
+Open Formulieren heeft een module om beslistabellen/beslissingen te evalueren. Deze
+dient :ref:`geconfigureerd <configuration_dmn_index>` te zijn voor je hier gebruik van
+kan maken.
+
+.. seealso:: Zie ook het voorbeeldformulier :ref:`examples_camunda`.
+
+Wat zijn beslistabellen?
+========================
+
+In beslistabellen kan je regels vastleggen - op basis van een aantal inputs en deze
+regels volgt hier een (berekend) resultaat uit. Deze tabellen worden door een
+*DMN-engine* uitgevoerd om het resultaat te bepalen. Beslistabellen zijn doorgaans
+een overzichtelijke, gecentraliseerde manier om regels (zoals wetgeving) te toetsen,
+en zijn vaak een goed alternatief voor complexe JsonLogic-expressies.
+
+Configuratie
+============
+
+DMN-acties moeten geconfigureerd worden.
+
+**Algemene instellingen**
+
+* Pluginkeuze
+* Beslisdefinitie/tabel om te evalueren
+* Eventuele versie van de beslissing - indien niet opgegeven, dan wordt de meest recente
```

Comment:
Maybe this should be part of the help text for the `Decision definition version` field as well?

Reply:
added - note that it doesn't render properly in storybook (at least in dev mode) because of the relative CSS icon URLs, so it's not visible there